src/code/z_parameter.c

```diff
@@ -2779,17 +2800,552 @@ s16 D_801BFB24[] = { 0, 255, 100, 0 };   // magicArrowEffectsB
+
+                            if (D_801BFBE4 == 0) {
+                                D_801BFBCC = D_801BFBEC[D_801BFBE8];
+                                D_801BFBD0 = D_801BFBF0[D_801BFBE8];
+                                D_801BFBD4 = D_801BFBF4[D_801BFBE8];
+                                D_801BFBD8 = D_801BFBF8[D_801BFBE8];
+                                D_801BFBDC = D_801BFBFC[D_801BFBE8];
+                                D_801BFBE0 = D_801BFC00[D_801BFBE8];
+                                D_801BFBE4 = 15;
+                                D_801BFBE8 ^= 1;
+                            }
+
+                            gDPSetCombineLERP(OVERLAY_DISP++, PRIMITIVE, ENVIRONMENT, TEXEL0, ENVIRONMENT, TEXEL0, 0,
+                                              PRIMITIVE, 0, PRIMITIVE, ENVIRONMENT, TEXEL0, ENVIRONMENT, TEXEL0, 0,
+                                              PRIMITIVE, 0);
+                            gDPSetPrimColor(OVERLAY_DISP++, 0, 0, D_801BFBCC, D_801BFBD0, 255, sThreeDayClockAlpha);
+                            gDPSetEnvColor(OVERLAY_DISP++, D_801BFBD8, D_801BFBDC, D_801BFBE0, 0);
+                        } else {
+                            // Clock diamond is green for regular timeSpeedOffset
+                            gDPSetCombineMode(OVERLAY_DISP++, G_CC_MODULATEIA_PRIM, G_CC_MODULATEIA_PRIM);
+                            gDPSetPrimColor(OVERLAY_DISP++, 0, 0, 0, 170, 100, sThreeDayClockAlpha);
+                        }
+
+                        OVERLAY_DISP = Gfx_DrawTexRectIA8(OVERLAY_DISP, gThreeDayClockDiamondTex, 40, 32, 140, 190, 40,
+                                                          32, 1 << 10, 1 << 10);
+
+                        /**
+                         * Draw Three-Day Clock's Day-Number over Diamond
+                         */
+                        gDPPipeSync(OVERLAY_DISP++);
+                        gDPSetPrimColor(OVERLAY_DISP++, 0, 0, 255, 255, 155, sThreeDayClockAlpha);
+
+                        OVERLAY_DISP = Gfx_DrawTexRectIA8(OVERLAY_DISP, interfaceCtx->doActionSegment + 0x780, 48, 27,
+                                                          137, 192, 48, 27, 1 << 10, 1 << 10);
+
+                        /**
+                         * Draw Three-Day Clock's Star (for the Minute Tracker)
+                         */
+                        gDPPipeSync(OVERLAY_DISP++);
+
+                        if (D_801BF974 != 0) {
+                            D_801BF980 += 0.02f;
+                            D_801BF97C += 11;
+                        } else {
+                            D_801BF980 -= 0.02f;
+                            D_801BF97C -= 11;
+                        }
+
+                        D_801BF978--;
+                        if (D_801BF978 == 0) {
+                            D_801BF978 = 10;
+                            D_801BF974 ^= 1;
+                        }
+
+                        sp1D0 = gSaveContext.save.time * 1.3183594f;
```

Comment:
Probably not worth a macro, but this constant looks like `TIME_TO_SECONDS_F` `((24.0f * 60.0f * 60.0f) / (0x10000))`

Reply:
I kind of like the idea of a `TIME_TO_SECONDS_F` macro, even if it only ends gets 1 use, because then it's available for use for other uses and completes the picture of time macros